data_prep/mp_prep_other_inputs_one_off.py

```diff
@@ -161,6 +239,51 @@ def mp_prep_other_inputs(tile_id_list, process):
         pool.close()
         pool.join()
 
+        for output_pattern in output_pattern_list:
+            processes = int(cn.count / 2)
+            uu.print_log("Checking for empty tiles of {0} pattern with {1} processors using light function...".format(output_pattern, processes))
+            pool = multiprocessing.Pool(processes)
+            pool.map(partial(uu.check_and_delete_if_empty_light, output_pattern=output_pattern), tile_id_list)
+            pool.close()
+            pool.join()
+
+        # Uploads output tiles to s3
+        if cn.NO_UPLOAD == False:
+            for i in range(0, len(output_dir_list)):
+                uu.upload_final_set(output_dir_list[i], output_pattern_list[i])
+
+    ##################################################################################################
+
+    if process == 'europe_forest' or process == 'all':
+
+        uu.print_log('Pre-processing European natural forest removal rate and standard deviation tiles')
+
+        # List of output directories and output file name patterns
+        output_dir_list = [
+            cn.annual_gain_AGC_BGC_natrl_forest_Europe_dir,
+            #cn.stdev_annual_gain_AGC_BGC_natrl_forest_Europe_dir   # raw files deleted
```

Comment:
Any idea why the Europe stdev output is commented out? I probably did that. Not likely to need to rerun, but please comment the stdev output back in unless I'm missing something here.

Reply:
Changes made in the following commit: 064f1d8211bb345593bb2eca6e424b5b216a7475